src/detect-engine-proto.c

```diff
@@ -143,6 +143,29 @@ int DetectProtoContainsProto(const DetectProto *dp, int proto)
     return 0;
 }
 
+/* List supported detect protocol keywords */
+
+void ListSupportedDetectProtocols(void)
+{
+    SCEnter();
+    printf("=========Supported Detect Protocols=========\n");
+	printf("- tcp\n");
```

Comment:
indent looks odd here

Reply:
Hmm need to double check in my vim as it looks ok there, but should be easy to fix.